button-merchant/src/main/java/com/usebutton/merchant/ButtonMerchant.java

```diff
@@ -77,7 +77,9 @@ public static void trackIncomingIntent(@NonNull Context context, @NonNull Intent
      *
      * @param order {@link Order}
      * @param userActivityListener {@link UserActivityListener}
+     * @deprecated
```

Comment:
We should include a link in the javadoc to the new `reportOrder()` method

Reply:
It will be done in the report order api request PR